HACKING.md

```diff
@@ -36,6 +36,17 @@ The docs can be generated with
 make doc
 ```
 
+To quickly get things up and running, you can also use docker/podman:
+
+```
+# build the image
+docker build -t papis .
+# to run the CI tests
+podman run -v $(pwd):/papis --rm -it papis
```

Comment:
I don't use docker or podman, so not sure what a good workflow would look like here. Why not just suggest one or the other, e.g.
```
docker run -it --rm papis 
```
?

Reply:
oops. I use podman on some machines and docker on others, but they are pretty much interchangable, so that is a copy-paste mistake!